cli/pack/systemd_dir.go

```diff
@@ -133,6 +133,9 @@ func getSystemdCtx(ctx *context.Ctx) *map[string]interface{} {
 	systemdCtx["AppEntrypointPath"] = project.GetAppEntrypointPath(ctx)
 	systemdCtx["StateboardEntrypointPath"] = project.GetStateboardEntrypointPath(ctx)
 
+	systemdCtx["FdLimit"] = project.GetFdLimit(ctx)
```

Comment:
What about filling default fd limit values on parsing a file into a structure?

Reply:
It can be like

```go
func getSystemdUnitParams(ctx) SystemdUnitParams {
  // checking and parsing systemdUnitParamsPath into systemdUnitParams

  checkAndSetDefaults(&systemdUnitParams.FdLimit, defaultInstanceFdLimit, func() {
    return systemdUnitParams.FdLimit >= minFdLimit
  })

  checkAndSetDefaults(&systemdUnitParams.StateboardFdLimit, defaultStateboardFdLimit, func() {
    return systemdUnitParams.StateboardFdLimit >= minStateboardFdLimit
  })
}
```